src/e3/testsuite/__init__.py

```diff
@@ -188,17 +188,26 @@ def testsuite_main(self, args: Optional[List[str]] = None) -> int:
         temp_group.add_argument(
             "-t", "--temp-dir", metavar="DIR", default=Env().tmp_dir
         )
+        temp_group.add_argument(
+            "--no-random-temp-subdir",
+            action="store_true",
+            help="Disable the creation of a random subdirectory in the"
+            " temporary directory. Use this when you know that you have"
+            " exclusive access to the temporary directory (needed in order to"
+            " avoid name clashes there) to get a deterministic path for"
+            " testsuite temporaries."
+        )
         temp_group.add_argument(
             "-d",
             "--dev-temp",
+            metavar="DIR",
             nargs="?",
             default=None,
             const="tmp",
-            help="Unlike --temp-dir, use this very directory to store"
-            " testsuite temporaries (i.e. no random subdirectory). Also"
-            " automatically disable temp dir cleanup, to be developer"
-            " friendly. If no directory is provided, use the local"
-            ' "tmp" directory',
+            help="Convenience shortcut for dev setups: forces `-t DIR"
+            " --no-random-temp-subdir --cleanup-mode=none` and cleanup `DIR`"
```

Comment:
"cleanup" -> "cleans up" (that's the only way I could understand this part of the sentence).

Reply:
Right, fixed